src/main/flight/gyroanalyse.c

```diff
@@ -118,16 +118,9 @@ static uint16_t FAST_DATA_ZERO_INIT   dynNotchMaxFFT;
 static float FAST_DATA_ZERO_INIT      smoothFactor;
 static uint8_t FAST_DATA_ZERO_INIT    numSamples;
 
-void gyroDataAnalyseInit(uint32_t targetLooptimeUs)
+void gyroDataAnalyseInit(gyroAnalyseState_t *state, uint32_t targetLooptimeUs)
 {
-#ifdef USE_MULTI_GYRO
```

Comment:
Looks like this code is entirely unneeded and a remnant from a time when `gyroDataAnalyseStateInit()` was called once for the initialisation of each gyro...